Presentation.Web/app/components/global-admin/global-admin-misc.controller.ts

```diff
@@ -135,12 +143,57 @@
                 } else {
                     notify.addInfoMessage("KLE opdatering stoppet!");
                 }
+            };
 
-            }
+            $scope.$watch("selectedUser", function (newVal, oldVal) {
+                if (newVal === oldVal || !newVal) return;
+
+                $scope.userOrganizations = [];
+                userService.getUserOrganizations(newVal).then(res => {
+                    $scope.userOrganizations.pushArray(res);
+                    $scope.userSelected = true;
+                });
+            });
+
+            $scope.removeUser = (id: number) => {
+                userService.deleteUser(id)
```

Comment:
remember to use the notifier to handle this. See above in the same file:

initial message should be:
Sletter NAME_AND_EMAIL_HERE

on success:
Brugeren Sletter NAME_AND_EMAIL_HERE er slettet

on error:
Fejl ifm. sletning af brugeren Sletter NAME_AND_EMAIL_HERE

Reply:
Do a state reload once deletion is finished to refresh these two:
![image](https://user-images.githubusercontent.com/13116889/171373382-68d40a13-5d90-4ac9-8823-70b8fb932897.png)